packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -671,60 +671,6 @@ describe('ReactDOMFloat', () => {
     );
   });
 
-  // @gate enableFloat
-  it('emits resources before everything else when rendering with no head', async () => {
```

Comment:
These tests weren't actually testing resource flushing order just hoistable order. These tests have been replaced by numerous others that actually cover the specific ordering so I've just removed them